src/test/java/org/broadinstitute/hellbender/utils/hdf5/HDF5PoNCreatorUnitTest.java

```diff
@@ -65,6 +69,36 @@ public void testCreateVariance() {
     }
 
     @Test
+    public void testCreateLargePoN(){
+        // final JavaSparkContext ctx = SparkContextFactory.getTestSparkContext();
+        // Creates a large PoN of junk values.
+        final int numEigenSamples = 20;
+
+//        // Load a lot of targets from a bed file
+//        final TargetCollection<? extends BEDFeature> inputTargetCollection = TargetUtils.readTargetFile(new File(BIG_TARGET_LIST));
+//        final List<Target> targetList = inputTargetCollection.targets().stream().map(TargetPadder::createTargetFromBEDFeature).collect(Collectors.toList());
+
+
+        // Make a big, fake set of read counts.
+        final RealMatrix bigCounts = new Array2DRowRealMatrix(3000000, 10);
```

Comment:
Maybe that big random matrix method I wanted you to extract shouldn't be private. . .